framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2651,7 +2651,7 @@ protected void parseStubFiles() {
      * @return the annotation mirror for anno
      */
     @Override
-    public AnnotationMirror getDeclAnnotation(Element elt,
+    public final AnnotationMirror getDeclAnnotation(Element elt,
```

Comment:
The three-argument getDeclAnnotation is currently private. Should we make it protected?